rest/exchange_create_loan.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2024-present Coinbase Global, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json, hmac, hashlib, time, requests, base64, os
+from urllib.parse import urlparse
+
+API_KEY = str(os.environ.get('API_KEY'))
+PASSPHRASE = str(os.environ.get('PASSPHRASE'))
+SECRET_KEY = str(os.environ.get('SECRET_KEY'))
+PROFILE_ID = str(os.environ.get('PROFILE_ID'))
+
+url = 'https://api.exchange.coinbase.com/loans/open'
+
+timestamp = str(int(time.time()))
+method = 'POST'
+
+url_path = urlparse(url).path
+
+payload = {
+   'loan_id': 'loan_id',
```

Comment:
Do we want working amounts here? Have you tested this endpoint? it lets you set the loan id externally? should that be generated as a uuid here?